README.md

```diff
@@ -96,3 +96,104 @@ So, if you want to distribute a Forge port of Iris, we'd prefer if you let us kn
 Iris is completely free and open source, and you are free to read, distribute, and modify the code as long as you abide by the (fairly reasonable) terms of the [GNU LGPLv3 license](https://github.com/IrisShaders/Iris/blob/master/LICENSE).
 
 Though it's not legally required, I'd appreciate it if you could ask before hosting your own public downloads for compiled versions of Iris. Though if you want to add the mod to a site like MCBBS, that's fine, no need to ask me.
+
+## FAQ
+> Q. How do I download Iris?
+
+A. Please visit the official Iris website: https://irisshaders.net/
+
+> Q. Why doesn't the Iris Installer work?
+
+A. You need java, please get it from: https://adoptium.net/?variant=openjdk17&jvmVariant=hotspot
+
+> Q. How can I configure my shaders?
+
+A. You can't currently, a configuration menu will be coming in a future version. (https://github.com/IrisShaders/Iris/issues/663)
```

Comment:
Can we not just put #663 here. since we're on GitHub?

Reply:
Yeah sure

Reply:
Done